docs/content/development/extensions-core/approximate-histograms.md

```diff
@@ -124,6 +124,7 @@ For general histogram and quantile use cases, the [DataSketches Quantiles Sketch
 |`upperLimit`|Upper limit of the histogram. |No default, must be specified|
 |`numBuckets`|Number of buckets for the histogram. The range [lowerLimit, upperLimit] will be divided into `numBuckets` intervals of equal size.|10|
 |`outlierHandlingMode`|Specifies how values outside of [lowerLimit, upperLimit] will be handled. Supported modes are "ignore", "overflow", and "clip". See [outlier handling modes](#outlier-handling-modes) for more details.|No default, must be specified|
+|`finalizeAsBase64Binary`|If true, the finalized aggregator value will be a Base64-encoded byte array containing the [serialized form](#serialization-formats) of the histogram. If false, the finalized aggregator value will be a JSON representation of the histogram.|false|
```

Comment:
Hmm, I actually sort of think this property should be flipped around since many other complex types will serialize as base64 by default, and the user should opt in to the friendly output. This would keep default behavior consistent for all complex types.

Reply:
The ApproximateHistogram in the same extension currently has a human-readable finalized form, so I decided to keep this false for consistency. I also added the same property to ApproximateHistogramFactory

Reply:
That makes sense, to at least keep it consistent within the same extension :+1: